source/03-specifications/03-counter-report-common-attributes-and-elements.rst

```diff
@@ -103,8 +103,14 @@ The table lists the Data_Types and related Host_Types which use them in one or m
 Report providers MUST report metrics in line with the following:
 
 * Host_Types required to provide the TR MUST deliver title level information in the PR (e.g. Journal for a journal article or Book for a book). If the DR is also required, usage MUST be reported at the title level within the DR.
-* Host_Types required to provide the TR and which choose to also offer the IR MUST report usage at the item level in IR (e.g. Article for a journal article, Book_Segment for a book).
-* Host_Types that are required to provide the IR MUST also report at item level in the PR and, if required, the DR. Only certain Data_Type and Parent_Data_Type combinations are permitted, as detailed in Table 3.q.
+* Investigations and Requests reported in the DR MUST be reported with the same Data_Type in the PR.
+* Investigations and Requests reported in the TR MUST be reported with the same Data_Type in the DR (if applicable) and the PR.
+* Investigations and Requests reported in the IR but not in the DR or TR MUST be reported with the same Data_Type in the PR.
+
+Parent_Data_Types apply where usage is reported in the IR and either the DR or TR. Only certain Data_Type and Parent_Data_Type combinations are permitted, as detailed in Table 3.q.
+
+* Usage reported in the IR that is also reported in the DR but not in TR MUST include the Data_Type from the DR as the Parent_Data_Type in the IR.
+* Usage reported in the IR that is also reported in the TR MUST include the Data_Type from the TR as the Parent_Data_Type in the IR, unless the Data_Type and Parent_Data_Type would be identical (in which case the Parent_Data_Type MUST be omitted).
```

Comment:
My idea was to replace the rules below Table 3.p with the new rules, but I hadn't noticed that there are also rules above Table 3.p that are also covered by the new rules. We now have the new Parent rules above and the old Parent rules below Table 3.p. I think it would be better to have all rules in one place, therefore I would suggest to remove the old Parent rules below Table 3.p.

Additionally I would suggest to remove the old "Host_Types required..." rule. What is required is covered now in the Host_Types section, and the rules should not depend on what is required, but on what is actually delivered. For example if a Repository decides to also provide TR, the Data_Types from TR should be used in PR (for consistency with other Host_Types), not the Data_Types from IR. This is already covered by the new rules.

I also noticed the reference to metrics in the sentence above the list, which seems wrong to me since there is nothing about metrics in this section.

Reply:
Actually the rules below Table 3.p cannot be removed, therefore I would suggest to merge the Parent rules section that is currently above Table 3.p with the rules below Table 3.p, so that all rules for the Parent are in one place. If that's okay, I can amend the pull request accordingly.